packages/publisher/src/generate-packages.ts

```diff
@@ -241,7 +246,6 @@ export function createNotNeededPackageJSON(
     main: "",
     scripts: {},
     author: "",
-    repository: registry === Registry.NPM ? sourceRepoURL : "https://github.com/types/_definitelytypedmirror.git",
```

Comment:
I think this is deleted because both npm and github deprecated packages get the project url from npm. Is that right?

I'm not actually sure why github packages have a weird default. My best guess: I must have thought the url pointed to npm when I was writing the code.